### ISSUE TITLE
Warning in log

### ISSUE THREAD

**Issue by emartynov** [slackhq/compose-lints#80]
Hi people, 

I migrated today to 1.1.0 and discovered a warning in the log:
```
Issue ComposeM2Api has defaultSeverity=IGNORE; that's not valid. Use enabledByDefault=false instead.
```

We are on `AGP 7.4.2`. And we run lint with `checkDependencies=true` (not sure the last one is important).

**Comment by ZacSweers** [comment]
Interesting, IGNORE is in lint's severity APIs 🤔. TIL about the alternative, found it and putting up a PR shortly

**Comment by ZacSweers** [comment]
Fixed in 1.1.1: https://github.com/slackhq/compose-lints/releases/tag/1.1.1. Thanks for the report!
